UnityProject/Assets/Scripts/UI/Systems/IngameMenu/LobbyUIPlayerListController.cs

```diff
@@ -41,9 +41,25 @@ public void GenerateList()
 			var c = playerEntry.GetComponent<LobbyUIListTemplate>();
 			c.playerName.text = $"{player.Tag} {player.UserName}";
 			c.playerNumber.text = i.ToString();
+			c.playerPing.text = player.PingToServer >= 0 ? $"{player.PingToServer}ms" : "NaN";
 			playerEntryList.Add(player, c);
-
 			playerEntry.transform.SetParent(playerTemplate.transform.parent, false);
 		}
 	}
+
+	public IEnumerator RefreshPing(GameObject list)
+	{
+		yield return WaitFor.Seconds(5f);
+		if (list.activeSelf)
+		{
+			UpdatePingList();
+			GenerateList();
+			StartCoroutine(RefreshPing(list));
+		}
+	}
+
+	private void UpdatePingList()
+	{
+		Messages.Server.UpdateConnectedPlayersMessage.Send();
+	}
```

Comment:
This refresh will only work on local host when they have the player list open

Reply:
Fixed 👍